Docs: added a note in kubectl-plugin docs to guide users when to use kubectl create workspace (#3560)
kcp-dev/kcp: docs/content/setup/kubectl-plugin.md

$ kubectl krew install kcp-dev/ws
$ kubectl krew install kcp-dev/create-workspace
```
!!! Note

>     When Installed directly from the repo we use `kubectl create workspace`
>     instead of `kubectl create-workspace`

Reviewer: ```suggestion
    When installed directly from the git repository `kubectl create workspace` should be used
    instead of `kubectl create-workspace`.
```
